setup.py

```diff
@@ -1,47 +0,0 @@
-#!/usr/bin/env python3
-
-# Copyright 2021-2022 NVIDIA Corporation
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-import os
-from pathlib import Path
-
-from setuptools import find_packages
-from skbuild import setup
-
-import legate.install_info as lg_install_info
-
-legate_dir = Path(lg_install_info.libpath).parent.as_posix()
-
-# ensure 'native' is used if CUDAARCHS isn't set
-# (instead of the CMake default which is a specific architecture)
-# ref: https://cmake.org/cmake/help/latest/variable/CMAKE_CUDA_ARCHITECTURES.html
-cuda_arch = os.getenv("CUDAARCHS", "native")
-
-cmake_flags = [
-    f"-Dlegate_ROOT:STRING={legate_dir}",
-    f"-DCMAKE_CUDA_ARCHITECTURES={cuda_arch}",
-]
```

Comment:
With this removed, will the following still work?

https://github.com/rapidsai/legate-boost/blob/ba062bf666845f82f5e8ab4690c97db4469ab42f/contributing.md#L47-L52

It might need to be updated to something like

```shell
SKBUILD_CMAKE_ARGS="-DCMAKE_CUDA_ARCHITECTURES='70;80'"
```

Or something similar.

Reply:
Good find! Pushing another commit to introduce `rapids_cuda_init_architectures(legateboost)`, that seems like a reasonable thing to me.  In `legate-raft` and `legate-dataframe`, I did use the "RAPIDS" architectures (but then, they use rapids libraries...)

The difference is that if you just do a plain `pip install .` RAPIDS seems to default to the `"RAPIDS"` architectures and not "native".
So there is a slightly unfortunate change, but at least this will keep working (maybe only with the new commit).

Reply:
Ah perfect, `rapids_cuda_init_architectures()` seems like the right way to do this (I did not actually know about that 🙈 )